### ISSUE TITLE
[Feature] File selection dialog: "Toggle Location Entry" button features as in default Nemo

### ISSUE THREAD

**Issue by danimesq** [linuxmint/nemo#3199]
With this, some users like me will be able to paste a file path right into that dialog.

**Comment by richardm1** [comment]
Agree. Windows gained this capability 16 years ago and it's a little jarring to have a path all ready to paste but nowhere to paste it.

**Comment by JosephMcc** [comment]
This issue has been brought up a ton of times if you search for it. The file selection dialog has nothing to do with Nemo and is out of Mint's control. That dialog is provided by whatever toolkit the app is built with.

**Comment by mgarcia-org** [comment]
how has this still not been fixed, such a simple request!

**Comment by JosephMcc** [comment]
> how has this still not been fixed, such a simple request!

Doesn't matter how simple it is. This has nothing to do with Mint or Nemo. In needs to go to the GTK devs.

**Comment by mgarcia-org** [comment]
I'm completely ignorant, I thought GTK is the API and nemo is the application, so not sure what it has to do with GTK?

**Comment by dominichayesferen** [comment]
This would actually make sense now as an 'issue' due to XDG Portals, although that'd be a Cinnamon thing.
